src/mocks/mock-sortOptions.ts

```diff
@@ -0,0 +1,18 @@
+import { SortOptionInterface } from '@shared/interfaces/sorting-interfaces';
+
+export enum SortTypeEnum {
+    asc = 'asc',
```

Comment:
I would suggest to have enums with number values.

```ts
asc = 1,
desc = 2

Reply:
Why? I understand better when I have enums with string, because later I have check for the sorting direction.

Reply:
It just a suggestion, you can skip it for now.
But remember, on real projects we you will always have enums with number values.

Reply:
On DB level it's better to store enum with number values.